### DISCUSSION TITLE
The live wallpaper will disappear when i playing the game in full screen mode? P/s: I use Laptop.

### DISCUSSION THREAD

**Discussion by CiaoBeiFeng** [rocksdanister/lively#1950] [Q&A]
Hello support team, my live wallpaper disappears when I play the game in full screen mode. I went back into Lively Wallpaper and used that wallpaper again and it just flashed white on the screen once and then stopped working. So why is that and please help me correct the error. Thank you.

**Comment by rocksdanister** [comment]
duplicate: https://github.com/rocksdanister/lively/issues/1926
